mqtt-sn-sub.c

```diff
@@ -131,7 +135,11 @@ static void parse_opts(int argc, char** argv)
             break;
 
         case 'v':
-            verbose = TRUE;
+            verbose = verbose == 0 ? 1 : verbose;
```

Comment:
Could you add some brackets to make it easier to read please?
Does this prevent -Vv setting verbose level back down to 1?